workspace/utilities/com/com-embed-video.xsl

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: com-embed-video -->
+<xsl:template name="com-embed-video">
+	<xsl:param name="thumbnail" select="thumbnail" />
+	<xsl:param name="is-autoplay" select="true()" />
+	<xsl:param name="autoload" select="'none'" />
+	<xsl:param name="title" select="''" />
+	<xsl:param name="ext-attr" />
+
+	<xsl:variable name="has-thumbnail" select="string-length($thumbnail) != 0" />
+
+
+	<xsl:variable name="attr">
+		<add class="padding-bottom-9_16 relative z-index-1" />
+		<add class="bg-color-black" />
+		<add data-playing-state-follower=".js-auto-oembed-play" />
+		<xsl:copy-of select="$ext-attr" />
+		<add dev-component="com-embed-video" />
+	</xsl:variable>
+
+	<xsl:variable name="attr-video">
+		<add class="absolute fill" />
+		<xsl:call-template name="util-oembed-attr" />
+		<add data-autoload="{$autoload}" />
+		<add data-rel="0" />
+		<xsl:if test="$is-autoplay = true()">
+			<add data-autoplay="1" />
+		</xsl:if>
+		<add class="js-auto-oembed-player" />
+		<add dev-element="video" />
+	</xsl:variable>
+
+	<xsl:variable name="attr-placeholder">
+		<add class="absolute fill overflow-hidden" />
+		<add class="cursor-pointer z-index-1" />
+		<add class="js-auto-oembed-play" />
+		<!-- STATE: playing -->
+		<add class="transition-opacity transition-duration-fast transition-ease-standard-out" />
+		<add data-playing-state-add-class="transparent pointer-events-none" />
+		<add dev-element="placeholder" />
+	</xsl:variable>
+
+	<xsl:variable name="attr-image">
+		<add class="absolute fill" />
+		<add class="bg-size-cover bg-position-center" />
+		<xsl:if test="$has-thumbnail">
+			<add style="background-image: url({$thumbnail});" />
```

Comment:
Thumbnail may not be an url.

Reply:
The goal is the use the oembed thumbnail, not one from the CMS. Do you think we should support both? Have a default fallback when there is no image (Like a black background or something)?

Reply:
@PascalPiche Proposed something: thumbnail always from the embed data. custom-thumbnail can come from the cms. @alexnantel88 @baptLesta @hotdoy  What do you think? Any ideas?

Reply:
I'm all in for the default thumbnail is from the provider. We did this is few projects and it works like a charm.

Reply:
Yeah, good, why not. But some providers to not five any thumbnail.

Reply:
> This is only applicable to vimeo/youtube. 

There are no guarantee that they will give it ! The code should handle that and not output crap we do not need.